spec/subjects/async-subject-spec.js

```diff
@@ -1,19 +1,23 @@
 var Rx = require('../../dist/cjs/Rx');
 
 var AsyncSubject = Rx.AsyncSubject;
-var nextTick = Rx.Scheduler.nextTick;
 var Observable = Rx.Observable;
 
 describe('AsyncSubject', function () {
+  var results = [];
```

Comment:
Definitely yes, avoid global state in tests, it's totally ok to repeat some common code in tests. The rule of thumb is: you should be able to copy a test and paste it in a discussion or in some execution environment, and everything you need for executing it is there available with no outer context assumed.